### ISSUE TITLE
StackOverflowException.

### ISSUE THREAD

**Issue by uzrptav** [erdomke/RtfPipe#34]
Hello! Please, Help! 
I'm trying to parse **RTF** file and get **StackOverflowException**.

```
var rtf = System.IO.File.ReadAllText("test.rtf");
var html = Rtf.ToHtml(rtf);
```
Example in attachment (please, rename `"test_RTF.txt"` ->`"test.rtf"` )

[test_RTF.txt](https://github.com/erdomke/RtfPipe/files/3697628/test_RTF.txt)

Thank you a lot!

**Comment by sanjay95** [comment]
try this 
  StreamReader file = new StreamReader("test.rtf");
                RtfSource source = file.ReadToEnd();
                var html = Rtf.ToHtml(source);

**Comment by uzrptav** [comment]
Hello!
Thnx a lot for reply, but it still does not  work.
I have *System.StackOverflowException* in *HtmlWriter *class in the
method *PxString(UnitValue
value) *on code line:

*var px = value.ToPx().ToString("0.#");*

вт, 15 окт. 2019 г. в 08:14, sanjay95 <notifications@github.com>:

**Comment by erdomke** [comment]
This should be fixed with version 1.0.7232.28557 which is being pushed to NuGet.
